io.github.accessory.minus_games_client.yml

```diff
@@ -0,0 +1,439 @@
+id: io.github.accessory.minus_games_client
+runtime: org.freedesktop.Platform
+runtime-version: &runtime-version '23.08'
+x-gl-version: &gl-version '1.4'
+x-gl-versions: &gl-versions 23.08;23.08-extra;1.4
+x-gl-merge-dirs: &gl-merge-dirs vulkan/icd.d;glvnd/egl_vendor.d;OpenCL/vendors;lib/dri;lib/d3d;vulkan/explicit_layer.d;vulkan/implicit_layer.d
+sdk: org.freedesktop.Sdk
+command: run.sh
+separate-locales: false
+sdk-extensions:
+  - org.freedesktop.Sdk.Compat.i386
+  - org.freedesktop.Sdk.Extension.toolchain-i386
+  - org.freedesktop.Sdk.Extension.rust-stable
+
+finish-args:
+  - --allow=devel
+  - --allow=multiarch
+  - --device=all
+  - --allow=bluetooth
+  - --allow=per-app-dev-shm
+  - --env=PATH=/app/bin:/app/utils/bin:/usr/bin:/usr/lib/extensions/vulkan/MangoHud/bin:/usr/lib/extensions/vulkan/gamescope/bin:/usr/lib/extensions/vulkan/OBSVkCapture/bin
+  - --filesystem=xdg-data/Steam:rw
+  - --filesystem=xdg-data/applications:rw
+  - --filesystem=~/.steam:rw
+  - --filesystem=~/Games:rw
+  - --filesystem=~/.local/share/Steam:rw
+  - --filesystem=~/.var/app/com.valvesoftware.Steam:rw
+  - --filesystem=~/.var/app/org.openwinecomponents.umu.umu-launcher:rw
```

Comment:
since you have `filesystem=home` these filesystem permissions are redundant.

Reply:
removed